orchestra/management/commands/loadworkflow.py

```diff
@@ -0,0 +1,35 @@
+from django.core.management.base import BaseCommand, CommandError
+from orchestra.workflow.exceptions import WorkflowError
+from orchestra.workflow.load import load_workflow
+
+import os
+
+
+class Command(BaseCommand):
+    help = 'Loads a workflow version into the database'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'workflow_directory',
+            help='The code directory containing the workflow.')
+        parser.add_argument(
+            'version_slug', nargs='?', default=None,
+            help=('(Optional) The version to load. If omitted, all available '
+                  'versions are loaded.'))
+        parser.add_argument(
+            '--force', action='store_true', default=False,
+            help=('If a workflow version already exists, update the existing '
+                  'version instead of exiting.'))
+
+    def handle(self, *args, **options):
+        workflow_dir = os.path.abspath(options['workflow_directory'])
+        if not os.path.exists(workflow_dir):
+            raise CommandError('Workflow directory does not exist.')
+
+        version_slug = options['version_slug']
+        versions = version_slug if version_slug is None else [version_slug]
```

Comment:
if it doesn't, a `WorkflowError` will be raised inside of `load_workflow`. Checking here would require duplicating some logic, I think.

Maybe the check for the workflow dir not existing (L. 26) should be moved inside `load_workflow` for consistency?